xbmc/utils/ArtUtils.cpp

```diff
@@ -158,11 +158,17 @@ std::string GetFolderThumb(const CFileItem& item, const std::string& folderJPG /
 {
   std::string strFolder = item.GetPath();
 
-  if (item.IsStack() || URIUtils::IsInRAR(strFolder) || URIUtils::IsInZIP(strFolder))
+  if (item.IsStack())
   {
     URIUtils::GetParentPath(item.GetPath(), strFolder);
   }
 
+  if (URIUtils::IsInRAR(strFolder) || URIUtils::IsInZIP(strFolder))
```

Comment:
Can it happen that an item is both "Stack" and "InRAR"/"InZIP" - I mean should this read `else if (URI...`?

Reply:
Yup. This is on purpose. E.g. two cd (yes, this is old school) rarred movies can be stacked as
stack://rar://somewhere/cd1.avi, rar.//somewhere/cd2.avi.

Reply:
Alright, thanks for the explanation.